lib/pager_duty/connection.rb

```diff
@@ -67,9 +67,11 @@ class ConvertTimesParametersToISO8601 < Faraday::Middleware
       def call(env)
 
         body = env[:body]
-        TIME_KEYS.each do |key|
-          if body.has_key?(key)
-            body[key] = body[key].iso8601 if body[key].respond_to?(:iso8601)
+        unless body.nil?
+          TIME_KEYS.each do |key|
+            if body.has_key?(key)
+              body[key] = body[key].iso8601 if body[key].respond_to?(:iso8601)
+            end
```

Comment:
This will skip the `@app.call` on line 78.  That's why I didn't use a guard clause, as I didn't want to duplicate that line of code.  Is that `@app.call env` line safe to skip in this case?  I'm not familiar with Faraday Middleware.

Reply:
Ah, right you are. I missed that at the bottom of the method. I still like guard clauses, but that can be a followup. Going to merge to be able to release this.